<launch>
  <param name="use_sim_time" value="true"/>

  <node pkg="box_detector" type="planar_patch_node.py" name="planar_patch" output="screen">
    <rosparam file="$(find box_detector)/config/params.yaml" command="load"/>
  </node>
</launch>
